supervisor/resolution/const.py

```diff
@@ -25,10 +35,15 @@ class IssueType(str, Enum):
     """Issue type."""
 
     FREE_SPACE = "free_space"
+    CORRUPT_DOCKER = "corrupt_docker"
+    MISSING_IMAGE = "missing_image"
+    UPDATE_FAILS = "update_fails"
```

Comment:
```suggestion
    UPDATE_FAILED = "update_failed"
```

Reply:
This is already in use, so needs to be updated elsewhere if the suggestion is accepted.